echidna/scripts/zero_nu_limit.py

```diff
@@ -23,56 +23,80 @@
 import echidna.limit.limit_config as limit_config
 import echidna.limit.limit_setting as limit_setting
 import echidna.limit.chi_squared as chi_squared
+import echidna.output.plot_chi_squared as plot_chi_squared
 
 
 if __name__ == "__main__":
+    import argparse
+    
+
+    parser = argparse.ArgumentParser(description="Example limit setting script")
+    parser.add_argument("-v", "--verbose", action="store_true",
+                        help="Print progress and timing information")
+    args = parser.parse_args()
+
     # Create signal spectrum
     Te130_0n2b = store.load(echidna.__echidna_home__ +
-                            "data/TeLoadedTe130_0n2b.ntuple_reco.hdf5")
+                            "/data/Te130_0n2b_mc_smeared.hdf5")
+    print Te130_0n2b._name
 
     # Create background spectra
     Te130_2n2b = store.load(echidna.__echidna_home__ +
-                            "data/TeLoadedTe130_2n2b.ntuple_reco.hdf5")
-    B8 = store.load(echidna.__echidna_home__ +
-                    "data/TeLoadedB8.ntuple_reco.hdf5")
+                            "/data/Te130_2n2b_mc_smeared.hdf5")
+    print Te130_2n2b._name
+    B8_Solar = store.load(echidna.__echidna_home__ +
+                          "/data/B8_Solar_mc_smeared.hdf5")
```

Comment:
Yeah good point. I think the original idea was to have an "approved" script, that worked with a specific set of spectra, but as we don't yet have a way of sharing/version controlling hdf5 files, perhaps your way would be more suitable for the moment.